src/commands/gif.ts

```diff
@@ -0,0 +1,52 @@
+import { logger } from "../logger";
```

Comment:
I dont see the need of this file when discord has `/giphy` build in and it does the same thing.

Reply:
I have further comments depending on if we keep this file or now

Reply:
Yeah, so the reason I created this command is that I saw some of the commands from other bots to retrieve kind of funny gifs with the keyword we rarely think of, I have also considered the `/giphy` build in so it depends on you.